<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="m_ShowTimeLabel">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>35</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_ShowDateLabel">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Label</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="m_YearComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="m_YearLabel">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Year</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_MonthComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="m_MonthLabel">
           <property name="text">
            <string>Month</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_DayComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="m_DayLabel">
           <property name="text">
            <string>Day</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="m_HourComboBox">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Hour</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_MinuteComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Minute</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_SecendsComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Second</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="m_StarButton">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_ZodiacButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/zodiac/ico/7.ico</normaloff>:/zodiac/ico/7.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="m_ToPreDayButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>前一天</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/operation/ico/left.ico</normaloff>:/operation/ico/left.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_NextDayButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>后一天</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/operation/ico/right.ico</normaloff>:/operation/ico/right.ico</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_TotodayButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>今天</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/operation/ico/today.ico</normaloff>:/operation/ico/today.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_DateSavePlushButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>SetDate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_TimePushButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>SetTime</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCalendarWidget" name="m_Calendar">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="navigationBarVisible">
        <bool>false</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
